flink/scripts/jobmanager-entrypoint.sh

```diff
@@ -0,0 +1,221 @@
+#! /bin/bash
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+set -e
+
+cobli_script="COBLI_ENTRYPOINT"
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+source ./common.sh
+
+submit_job() {
+  log_warn "> Submiting job..."
+  ensure_flink_config
+  /opt/flink/bin/standalone-job.sh "$@" 1>&2 &
+  log_warn "> Job submited"
+}
+
+ensure_ha_directories() {
+  log_warn "> Ensuring directories existence..."
+  mkdir -p "$@"
+  log_warn "> Directories present"
+}
+
+get_savepoint_ref() {
+  local savepoint_ref_path savepoint_path
+  savepoint_ref_path="$1"
+  savepoint_path=''
+  if [ -f "$savepoint_ref_path" ]; then
+    log_warn "> Savepoint reference file found"
+    log_warn "> Geting savepoint path..."
+    savepoint_path=$(sed 's/#.*//g' < "$savepoint_ref_path")
+    log_warn "> Got savepoint: <$savepoint_path>"
+  else
+    log_warn "> Savepoint file not fount"
+  fi
+  echo "$savepoint_path"
+}
+
+clean_zookeeper_data() {
+  local zk_host zk_root_path job_name
+  zk_host="$1"
+  zk_root_path="$2"
+  job_name="$3"
+
+  local zk_clean_cmd
+  zk_clean_cmd="rmr $zk_root_path/$job_name"
+
+  local zk_cp_0 zk_cp_1 zk_cp_2 zk_cp
+  zk_cp_0="lib/org.apache.zookeeper.zookeeper-3.4.13.jar"
+  zk_cp_1="lib/org.slf4j.slf4j-api-1.7.26.jar"
+  zk_cp_2="org.apache.zookeeper.ZooKeeperMain"
+  zk_cp="$zk_cp_0:$zk_cp_1 $zk_cp_2"
+
+  log_warn "> Cleaning zookeeper path: <$zk_root_path/$job_name>"
+
+  echo "$zk_clean_cmd" | java -cp $zk_cp -server $zk_host
```

Comment:
`org.apache.zookeeper.ZooKeeperMain` is the main class, not classpath. We could use:
```suggestion
  local zk_cp_0 zk_cp_1 zk_main_class
  zk_cp_0="lib/org.apache.zookeeper.zookeeper-3.4.13.jar"
  zk_cp_1="lib/org.slf4j.slf4j-api-1.7.26.jar"
  zk_cp="$zk_cp_0:$zk_cp_1"

  zk_main_class="org.apache.zookeeper.ZooKeeperMain"

  log_warn "> Cleaning zookeeper path: <$zk_root_path/$job_name>"

  echo "$zk_clean_cmd" | java -cp $zk_cp  $zk_main_class -server $zk_host
```

Reply:
Sorry, I aready fix that part. Could you check it again?